package.json

```diff
@@ -13,16 +13,16 @@
     "test:watch": "jest --watchAll"
   },
   "dependencies": {
-    "azure-iothub": "^1.2.5",
-    "azure-iothub-receiver": "^2.0.0",
-    "babel-runtime": "^6.26.0",
-    "chart.js": "^2.7.2",
-    "dotenv": "^5.0.0",
-    "express": "^4.12.4",
-    "marked": "^0.4.0",
+    "azure-iothub": "~1.2.5",
+    "azure-iothub-receiver": "~2.0.0",
+    "babel-runtime": "~6.26.0",
+    "chart.js": "~2.7.2",
+    "dotenv": "~5.0.0",
+    "express": "~4.16.0",
+    "marked": "~0.4.0",
     "socket.io": "*",
```

Comment:
I'm not 100% sure why just allow star on this one.

Reply:
So I've had some oddness with this one where I've used `2.1.1` and its crapped the bed (I wanna say that it got stuck on `UWS`. I can play around with it again as I didn't spend _much_ time trying to sort that one.